core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -23,20 +23,25 @@
     </div>
   </div>
   <div class="oppia-content-wrapper">
-    <button class="btn oppia-audio-button oppia-audio-bar-button-transition"
+    <button class="btn oppia-audio-button oppia-audio-bar-button-transition protractor-test-translation-start-record-accessibility"
             uib-tooltip="Record"
             tooltip-placement="above"
             ng-click="checkAndStartRecording()"
-            ng-disabled="!canTranslate || cannotRecord">
+            ng-disabled="!canTranslate || cannotRecord"
+            aria-label="Start Recording"
+            role="button"
```

Comment:
Do we need role over here? (As this is already a button tag.)

Reply:
No it was my mistake